package.json

```diff
@@ -4,7 +4,7 @@
     "test": "meteor test --once --driver-package dispatch:mocha-phantomjs",
     "test-watch": "meteor test --driver-package practicalmeteor:mocha",
     "test-app-watch": "meteor test --full-app --driver-package practicalmeteor:mocha",
-    "lint": "eslint .",
+    "lint": "meteor node node_modules/eslint/bin/eslint.js .",
```

Comment:
Ah I see what you mean I just tried it on boilerplate

    "gulp": "gulp",
    "lint": "gulp lint",
    "tidy": "gulp tidy"

Seems to work well even after `npm uninstall -g gulp` (I verified that `gulp` command itself will fail`)